tests/unit_tests/config/test_queue_config.py

```diff
@@ -30,6 +30,36 @@ def test_stop_long_running_is_set_from_corresponding_keyword(value):
     assert QueueConfig(stop_long_running=value).stop_long_running == value
 
 
+@pytest.mark.parametrize("queue_system", ["LSF", "TORQUE", "SLURM"])
+def test_project_code_is_set_when_forward_model_contains_simulator(queue_system):
```

Comment:
`.._contains_selected_simulators` ?? (since "simulator" is not a well-defined term)